packages/components/src/components/select/style.scss

```diff
@@ -1,6 +1,6 @@
 @import '../@shared/mixins';
+@import '../style';
```

Comment:
Diese ganzen Änderungen in der Reihenfolge sind nur notwendig, weil wir Mixin-Module haben, die auch Styles definieren - richtig?

`_kol-alert-mixin.scss` z.B. bindet style und host-display-block mit ein. Das halte ich hier für den eigentlichen Fehler, das sollte nicht passieren. Mixin-Module sollten keine Side-Effects haben.
Könnten wir das noch korrigieren, gerne auch in einem Follow Up?

Reply:
Reused CSS aufräumen, sollte ein eigenes Thema sein. Ich sehe das aber genauso.

Reply:
Hierfür machen wir ein Folgeticket: https://github.com/public-ui/kolibri/issues/7067